aws/aws-how-to/instances/automatically-setup-multiple-nics.rst

```diff
@@ -0,0 +1,55 @@
+Set up automated network configuration for instances with multiple NICs
+=======================================================================
+
+Ubuntu instances running on AWS can be configured to automatically configure source routing on multiple NICs via cloud-init, starting with `v24.1`_, on different `events`_, as:
```

Comment:
the v24.1 cloud-init link seems to be broken. 
Also, are these three the only events or is boot_legacy also an option? This is just a clarification that I need.

Reply:
v24.1 is not yet released, so that's why the link is broken. It is scheduled to be released this week.

BOOT_LEGACY is as BOOT (subsequent boots), but fetching and applying the config twice (at init-local and init timeframes), but being this a legacy / deprecated option, I did think it was better to not include it here, advanced user need that mode can connect the dots from here.

Reply:
Thanks, that makes sense.